src/com/google/enterprise/gsafeed/GsafeedHelper.java

```diff
@@ -0,0 +1,125 @@
+package com.google.enterprise.gsafeed;
+
+import org.w3c.dom.Document;
+import org.xml.sax.EntityResolver;
+import org.xml.sax.ErrorHandler;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+import org.xml.sax.SAXParseException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.StringReader;
+import java.net.URL;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+/**
+ * Helper methods for reading and creating GSA feed files.
+ */
+public class GsafeedHelper {
```

Comment:
The DTD uses multi-world names with no distinction, e.g., "gsafeed". Do we want to use the correct camelcase in our code, even if it doesn't match? Or do we want to change the generated code to match the style guide?

Reply:
I should say, I prefer changing everything to use the correct camelcase, per the style guide. I think GsaFeed is better than Gsafeed.